apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -837,17 +847,21 @@ func (p *ProvisionerAPI) processEachContainer(args params.Entities, handler perC
 		// The auth function (canAccess) checks that the machine is a
 		// top level machine (we filter those out next) or that the
 		// machine has the host as a parent.
-		container, err := p.getMachine(canAccess, machineTag)
+		ctr, err := p.getMachine(canAccess, machineTag)
```

Comment:
ctr unfortunately looks more like 'counter' to me than 'container'. Maybe its ok. I'm guessing this was done because of a collision with a package import?
Maybe we could use the term 'guest' or maybeContainer, or ?

Reply:
Yes, import collision. Happy to change it.